### PR TITLE
pci-dss appendix/article

### PR DESCRIPTION
**Pull request by sknorr**

### CODE REVIEW

File: xml/security_pcidss.xml [hunk 1]
```diff
@@ -7,272 +7,1251 @@
...
+      </term>
+      <listitem>
+       <para>
+        The guidance description of the &pcidssa; document says the following
+        about this requirement: <quote>Full disk encryption helps to protect
+        data in the event of physical loss of a disk and therefore may be
+        appropriate for portable devices that store cardholder data.</quote>
+       </para>
+       <para>
+        From the administrator point of view a block device encryption with
```
Comment (dmpop) [inline xml/security_pcidss.xml]: From the administrator point of view,

File: xml/security_pcidss.xml [hunk 2]
```diff
@@ -7,272 +7,1251 @@
...
+        the Linux Unified Key Setup (LUKS)/dm-crypt offers an abstraction
+        layer that allows the usage of fully encrypted devices in the same way
+        as non encrypted disks. Therefore access control can only be limited
+        with the general ACL permissions that the used file system offers. To
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>used</del> given

File: xml/security_pcidss.xml [hunk 3]
```diff
@@ -7,272 +7,1251 @@
...
+        be compliant with this requirement the used decryption key cannot be
```
Comment (dmpop) [inline xml/security_pcidss.xml]: be compliant with this requirement, the used

File: xml/security_pcidss.xml [hunk 4]
```diff
@@ -7,272 +7,1251 @@
...
+        associated with any general login credentials or authentication
+        methods. With LUKS this is usually the case as the password needs to
```
Comment (dmpop) [inline xml/security_pcidss.xml]: This is particularly important when using LUKS, as

File: xml/security_pcidss.xml [hunk 5]
```diff
@@ -7,272 +7,1251 @@
...
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>
+       3.4.1.c Examine the configurations and observe the processes to verify
+       that cardholder data on removable media is encrypted wherever stored.
+      </term>
+      <listitem>
+       <para>
+        As described in the last section, LUKS/dm-crypt is a full disk
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>last</del> previous

File: xml/security_pcidss.xml [hunk 6]
```diff
@@ -7,272 +7,1251 @@
...
+      </listitem>
+     </varlistentry>
+    </variablelist>
+   </sect3>
+   <sect3>
+    <title>Requirement 4: Encrypt Transmission of Cardholder Data Across Open, Public Networks</title>
+    <para>
+     Cardholder data needs to be encrypted during transmissions over insecure
+     networks. Best practice would be to try to encrypt all traffic,
+     externally and internally, to make it as hard as possible for malicious
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>malicious individuals</del> unauthorized parties

File: xml/security_pcidss.xml [hunk 7]
```diff
@@ -7,272 +7,1251 @@
...
+       TLS, IPSEC, SSH, etc.) to safeguard sensitive cardholder data during
+       transmission over open, public networks, including the following: (1)
+       Only trusted keys and certificates are accepted, (2) The protocol in
+       use only supports secure versions or configurations, (3) The
+       encryption strength is appropriate for the encryption methodology in
+       use.
+      </term>
+      <listitem>
+       <para>
+        Every connection that transmits sensitive information needs to be
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>Every</del> Any

File: xml/security_pcidss.xml [hunk 8]
```diff
@@ -7,272 +7,1251 @@
...
+        &productname; comes with a set of services and tools that allow
+        protected HTTPS connections. For example, this can be directly done
+        with the Apache HTTP Server or via <command>stunnel</command> that
+        functions as a proxy to offer TLS encryption functionality.
+       </para>
+       <para>
+        IPsec or other VPN technologies can be used for securing the
+        connection between network segments that are connected via a public
+        network. Such connections can also be secured with a public x.509
+        certificate. For internal usage it would be possible to use a private
```
Comment (dmpop) [inline xml/security_pcidss.xml]: For internal usage,

File: xml/security_pcidss.xml [hunk 9]
```diff
@@ -7,272 +7,1251 @@
...
+        Certificate Authority (CA) to sign x.509 certificates and to keep
+        track on trusted keys.
+       </para>
+       <para>
+        In &productname; this can be directly established with
```
Comment (dmpop) [inline xml/security_pcidss.xml]: In &productname;**,**

File: xml/security_pcidss.xml [hunk 10]
```diff
@@ -7,272 +7,1251 @@
...
+        <phrase role="productname">strongSwan</phrase> that is a IPsec-based
+        VPN solution or with OpenVPN that is using a custom security protocol.
+       </para>
+       <para>
+        Administration access to the OS level is usually done via SSH.
+        Configuration and authentication can be tightened to meet the needed
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>to meet the needed</del> to achieve the required level

File: xml/security_pcidss.xml [hunk 11]
```diff
@@ -7,272 +7,1251 @@
...
+        security. This was partially stated before inside the requirements 1
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>This was partially stated before inside</del> This is briefly discussed in

File: xml/security_pcidss.xml [hunk 12]
```diff
@@ -7,272 +7,1251 @@
...
+    </variablelist>
+   </sect3>
+  </sect2>
+  <sect2>
+   <title>Maintain a Vulnerability Management Program</title>
+   <para/>
+   <sect3>
+    <title>Requirement 5: Protect All Systems Against Malware and Regularly Update Anti-Virus Software or Programs</title>
+    <para>
+     For &pcidssa; compliance it is necessary to protect against malicious
```
Comment (dmpop) [inline xml/security_pcidss.xml]: For &pcidssa; compliance,

Comment (dmpop) [inline xml/security_pcidss.xml]: <del>malicious</del>

File: xml/security_pcidss.xml [hunk 13]
```diff
@@ -7,272 +7,1251 @@
...
+     software threats. Commercial anti-virus software is available from the
+     major anti-virus software vendors and can be integrated into the Linux
+     environment. &productname; itself comes with the open source anti-virus engine
+     &clamav;.
+    </para>
+    <para>
+     The question here is which functionality and performance is needed.
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>The question here is which functionality and performance is needed.</del> But you need to consider  specific functionality and performance.

File: xml/security_pcidss.xml [hunk 14]
```diff
@@ -7,272 +7,1251 @@
...
+     &clamav; has a limited set of scanning capabilities and limited
+     performance compared to commercial products. So the expectations need to
+     be clear that &clamav; only gives only a simple protection level.
+    </para>
+    <para>
+     On the other hand &clamav; is directly shipped with &productname; and can be
```
Comment (dmpop) [inline xml/security_pcidss.xml]: On the other hand &clamav;**,**

<del>directly</del>

with &productname;**,** and **it** can

File: xml/security_pcidss.xml [hunk 15]
```diff
@@ -7,272 +7,1251 @@
...
+     included during server installation. This makes it easy to fulfill the
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>This makes it easy to fulfill the details of this requirement</del> This allows you to meet this requirement

File: xml/security_pcidss.xml [hunk 16]
```diff
@@ -7,272 +7,1251 @@
...
+     details of this requirement, but the drawbacks compared to commercial
+     products need to be clearly understood.
+    </para>
+   </sect3>
+   <sect3>
+    <title>Requirement 6: Develop and maintain secure systems and applications</title>
+    <para>
+     A powerful instrument in &productname; is the tool
```
Comment (dmpop) [inline xml/security_pcidss.xml]: A powerful instrument in &productname; is the tool <command>zypper</command> that is an interface to

<command>zypper</command>  is a powerful tool that provides interface to

File: xml/security_pcidss.xml [hunk 17]
```diff
@@ -7,272 +7,1251 @@
...
+     <command>zypper</command> that is an interface to the ZYpp system management library
+     (libzypp). Among other things, it solves<!--fixme: wording--> packages, products, patterns and
```
Comment (dmpop) [inline xml/security_pcidss.xml]: it resolves <del>packages, products, patterns</del> dependencies

File: xml/security_pcidss.xml [hunk 18]
```diff
@@ -7,272 +7,1251 @@
...
+     patches dependencies, has a locking mechanism to prevent package
+     installation and provides a complete update stack to keep the system up
```
Comment (dmpop) [inline xml/security_pcidss.xml]: installation,

File: xml/security_pcidss.xml [hunk 19]
```diff
@@ -7,272 +7,1251 @@
...
+     to date and protected against know security issues. <command>zypper</command> is
+     part of any &productname; installation and has direct access to the update
+     repositories after system registration.
+    </para>
+    <para>
+     For system management we provide &susemgr; that provides an efficient
```
Comment (dmpop) [inline xml/security_pcidss.xml]: For system management,

<del>that provides</del> that offers

File: xml/security_pcidss.xml [hunk 20]
```diff
@@ -7,272 +7,1251 @@
...
+     way to keep systems up-to-date. It offers seamless management of both
+     &productname; and &rhel; client systems. This
+     is particularly handy in larger system environments, to check the
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>handy</del> useful

Comment (dmpop) [inline xml/security_pcidss.xml]: <del>to check</del> when you need to check

File: xml/security_pcidss.xml [hunk 21]
```diff
@@ -7,272 +7,1251 @@
...
+     current update status of each system and to react on know security risks.
```
Comment (dmpop) [inline xml/security_pcidss.xml]: know? known?

File: xml/security_pcidss.xml [hunk 22]
```diff
@@ -7,272 +7,1251 @@
...
+    </para>
+    <para>
+     Most parts of this requirement are about in-house security software
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>are about </del> concern

File: xml/security_pcidss.xml [hunk 23]
```diff
@@ -7,272 +7,1251 @@
...
+     development, documentation and design questions. In this prospect we
```
Comment (dmpop) [inline xml/security_pcidss.xml]: This document focuses on

File: xml/security_pcidss.xml [hunk 24]
```diff
@@ -7,272 +7,1251 @@
...
+       three months).
+      </term>
+      <listitem>
+       <para>
+        To identify patches that
+        need to be installed to secure your system, do the following:
+       </para>
+       <!-- FIXME: procedure -->
+       <para>
+        First refresh all software repositories so that we have up-to-date information.
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>First</del>

so the information is up to date.

File: xml/security_pcidss.xml [hunk 25]
```diff
@@ -7,272 +7,1251 @@
...
+       </para>
+       <screen>zypper refresh</screen>
+       <para>
+        Now we can look for important security fixes that have not yet been
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>Now we can look</del> Search for

File: xml/security_pcidss.xml [hunk 26]
```diff
@@ -7,272 +7,1251 @@
...
+        installed:
+       </para>
+       <screen>zypper list-patches --category security --severity important</screen>
+       <para>
+        It is also possible to search for CVE or SUSE Bugzilla numbers. By
+        default only needed patches are listed by this command. To show also
```
Comment (dmpop) [inline xml/security_pcidss.xml]: By default,

File: xml/security_pcidss.xml [hunk 27]
```diff
@@ -7,272 +7,1251 @@
...
+        patches that have already been installed use <option>--all</option>:
```
Comment (dmpop) [inline xml/security_pcidss.xml]: installed, use

File: xml/security_pcidss.xml [hunk 28]
```diff
@@ -7,272 +7,1251 @@
...
+       </para>
+       <screen>zypper list-patches --all --cve=CVE-2016-4957</screen>
+       <para>
+        Details of individual patches can be listed with the patch-info
+        subcommand:
+       </para>
+       <screen>zypper patch-info SUSE-SLE-SERVER-12-SP1-2016-600</screen>
+       <para>
+        To install only important security patches use the patch command. All
```
Comment (dmpop) [inline xml/security_pcidss.xml]: patches, use

File: xml/security_pcidss.xml [hunk 29]
```diff
@@ -7,272 +7,1251 @@
...
+        <command>zypper</command> commands can be run with the <option>--non-interactive</option> to run
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>can be run</del> can be executed using

File: xml/security_pcidss.xml [hunk 30]
```diff
@@ -7,272 +7,1251 @@
...
+   </sect3>
+  </sect2>
+  <sect2>
+   <title>Implement Strong Access Control Measures</title>
+   <para/>
+   <sect3>
+    <title>Requirement 7: Restrict access to cardholder data by business need to know</title>
+    <para>Summary:</para>
+    <para>
+     Access control is a complex topic if it comes to the operating system.
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>if it comes</del> when it concerns

File: xml/security_pcidss.xml [hunk 31]
```diff
@@ -7,272 +7,1251 @@
...
+     Again, the &pcidssa; requirement is quite fuzzy at this point and does not
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>is quite fuzzy</del> is not currently specified precisely <del>at this point</del>

File: xml/security_pcidss.xml [hunk 32]
```diff
@@ -7,272 +7,1251 @@
...
+     specifically state to which degree the restriction needs to be fulfilled.
+     &productname; comes with all general Linux tools to limit and
+     restrict access to certain system areas and components.
+    </para>
+    <para>
+     Access can be controlled via specific users and groups of users. This
+     are the traditional Unix permission settings. A more flexible mechanism
+     for file systems are Access Control Lists (ACLs) that offer a more
+     graduated approach. &selnx; can be used to separate the system to a
+     maximum and to protect processes from gaining more resources and access
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>can be used to separate the system to a maximum</del> for maximum system separation?

File: xml/security_pcidss.xml [hunk 33]
```diff
@@ -7,272 +7,1251 @@
...
+     then allowed.
+    </para>
+    <para>
+     &selnx; and &aa; are not within the scope of this document. However, to
+     protect critical systems that are likely to be targeted, research the
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>research</del> take a look at the existing

File: xml/security_pcidss.xml [hunk 34]
```diff
@@ -7,272 +7,1251 @@
...
+    </para>
+    <variablelist>
+     <varlistentry>
+      <term>
+       7.1.2 Restrict access to privileged user IDs to least privileges
+       necessary to perform job responsibilities.
+      </term>
+      <listitem>
+       <para>
+        The standard Unix permissions allow the setting of Read, Write and
```
Comment (dmpop) [inline xml/security_pcidss.xml]: The standard Unix permissions let you set Read, Write and Execution flags to user and group IDs.

File: xml/security_pcidss.xml [hunk 35]
```diff
@@ -7,272 +7,1251 @@
...
+        Execution flags for user and group IDs. A general group called
+        <systemitem class="groupname">others</systemitem> or
+        <systemitem class="groupname">world</systemitem> defines the access
+        for users that do not fit into
+        the first two groups. This allows a straightforward way to grant or
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>allows</del> provides

File: xml/security_pcidss.xml [hunk 36]
```diff
@@ -7,272 +7,1251 @@
...
+        deny access to file system resources.
+       </para>
+       <para>
+        ACLs allow an extra level of restriction. It is possible to set
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>allow</del> provide

File: xml/security_pcidss.xml [hunk 37]
```diff
@@ -7,272 +7,1251 @@
...
+        read-write access for one user ID and only read access to a second
+        one. The same goes for group IDs. The command-line tools are included
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>are included inside</del> supplied with

File: xml/security_pcidss.xml [hunk 38]
```diff
@@ -7,272 +7,1251 @@
...
+       <para>
+        Standard Unix permissions come also with a so called StickyBit. This
+        allows the execution of certain programs with higher privileges then
+        the user who is executing those programs. The best example for this
+        is the <command>passwd</command> tool that needs to modify the
+        <filename>/etc/shadow</filename> to
+        change the user's password.
+       </para>
+       <para>
+        In respect to the StickyBit Linux capabilities are also a way to
```
Comment (dmpop) [inline xml/security_pcidss.xml]: In respect to the StickyBit, Linux capabilities also offer a way

File: xml/security_pcidss.xml [hunk 39]
```diff
@@ -7,272 +7,1251 @@
...
+        explicitly allow certain operations or behaviors to binaries. An
+        example for this is the command <command>ping</command> from the
```
Comment (dmpop) [inline xml/security_pcidss.xml]: that is a part of the <package>iputils</package> package

File: xml/security_pcidss.xml [hunk 40]
```diff
@@ -7,272 +7,1251 @@
...
+        package <package>iputils</package>. It needs
```
Comment (dmpop) [inline xml/security_pcidss.xml]: While it doesn't need StickyBit, it needs

File: xml/security_pcidss.xml [hunk 41]
```diff
@@ -7,272 +7,1251 @@
...
+    <title>Requirement 8: <systemitem>Identify and authenticate access to system component</systemitem></title>
+    <para>
+     It is best practice to have a central database with user information and
+     a unique identifier (UID) to grant or deny access to certain system
+     components. This makes it easy to give administrators special access to
+     group of servers or a database engineer permission to a certain DBMS
+     system.
+    </para>
+    <para>
+     On a stand-alone server unique identifiers are already managed via the
```
Comment (dmpop) [inline xml/security_pcidss.xml]: On a stand-alone server,

File: xml/security_pcidss.xml [hunk 42]
```diff
@@ -7,272 +7,1251 @@
...
+     <filename>/etc/passwd</filename> and <filename>/etc/group</filename>.
+    </para>
+    <variablelist>
+     <varlistentry xml:id="vle.pcidss.disable-user">
+      <term>
+       8.1.4 Remove/disable inactive user accounts within 90 days.
+      </term>
+      <listitem>
+      <para>
+       In this context, There are many advantages to using a centralized
```
Comment (dmpop) [inline xml/security_pcidss.xml]: In this context, there are many

File: xml/security_pcidss.xml [hunk 43]
```diff
@@ -7,272 +7,1251 @@
...
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>
+       8.1.7 Set the lockout duration to a minimum of 30 minutes or until an
+       administrator enables the user ID.
+      </term>
+      <listitem>
+      <para>
+       The PAM module <systemitem>pam_tally2</systemitem> that was described
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>that was</del> described

File: xml/security_pcidss.xml [hunk 44]
```diff
@@ -7,272 +7,1251 @@
...
+       for <xref linkend="vle.pcidss.lock-user"/> can be used to lock an
+       account for a given time after a failed login attempt. The parameter
+       <literal>unlock_time=1800</literal> must be configured into the PAM
```
Comment (dmpop) [inline xml/security_pcidss.xml]: must be specify in

File: xml/security_pcidss.xml [hunk 45]
```diff
@@ -7,272 +7,1251 @@
...
+       configuration. The default setting is that only the administrator can
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>The default setting is that</del> By default,

File: xml/security_pcidss.xml [hunk 46]
```diff
@@ -7,272 +7,1251 @@
...
+     </varlistentry>
+     <varlistentry xml:id="vle.pcidss.multi-factor-local">
+      <term>
+       8.3.1 Incorporate multi-factor authentication for all non-console access
+       into the CDE for personnel with administrative access.
+      </term>
+      <listitem>
+       <para>
+        To authenticate users for administrative access with multiple factors, use
+        the pluggable authentication module (PAM) mechanism. This makes it
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>This makes it flexible to add</del> This add flexibility when adding

File: xml/security_pcidss.xml [hunk 47]
```diff
@@ -7,272 +7,1251 @@
...
+       <para>
+        Another way of multi-factor authentication for SSH connections is the
+        additional usage of public keys. To connect to a system, it is
+        necessary to prove that you are in the possession of the appropriate
+        private key. When accepted you need to enter a password at the second
+        stage. Thus, attackers need to acquire a private key before they can
+        even try to brute force a password prompt.
+       </para>
+       <para>
+        For commercial one-time password (OTP) products, there is usually
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>For commercial one-time password (OTP) products, there is usually also a Linux PAM module available.</del> A Linux PAM module is usually available for commercial one-time password (OTP) products.

File: xml/security_pcidss.xml [hunk 48]
```diff
@@ -7,272 +7,1251 @@
...
+    </para>
+   </sect3>
+  </sect2>
+  <sect2>
+   <title>Maintain an Information Security Policy</title>
+   <para/>
+   <sect3>
+    <title>Requirement 12: Maintain a Policy That Addresses Information Security for All Personnel</title>
+    <para>
+     Any organization that handles information wort
```
Comment (dmpop) [inline xml/security_pcidss.xml]: <del>information worthy of protection</del> valuable information

File: xml/security_pcidss.xml [hunk 49]
```diff
@@ -7,272 +7,1251 @@
...
+ renamed to NetIQ Sentinel at some point):
+ + https://www.netiq.com/industries/federal/regulations-standards-compliance/pci-dss.html
+ -->
+
+ <para>
+  An essential part of implementing &pcidssa; is the combination of actions.
+  It is not only necessary
+  to configure a service to be secure, but it is also important that the
+  configuration is not changed afterward. If something was changed, it is
+  important to keep track on who changed what at which point in time.
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/security_pcidss.xml]: fxd

File: xml/security_pcidss.xml [hunk 50]
```diff
@@ -7,272 +7,1251 @@
...
+  This document should give you a basic understanding of how
+  &productname; can be configured to comply with the &pcidss;.
+ </para>
  <para>
-  If credit or debit card data is processed at your site, it can be
-  necessary to follow the Payment Card Industry Data Security Standard
-  (PCI-DSS). This appendix will help you implement the portions of the
-  standard that relate to the operating system.
-<!-- and network configuration. -->
+  Again, it is most important to understand that protecting the systems does
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/security_pcidss.xml]: fxd

File: xml/security_pcidss.xml [hunk 51]
```diff
@@ -7,272 +7,1251 @@
...
+  not end by the configuration. It is the whole environment and people
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/security_pcidss.xml]: fxd

File: xml/security_pcidss.xml [hunk 52]
```diff
@@ -7,272 +7,1251 @@
...
+    <listitem>
+     <orderedlist>
+      <listitem override="3">
+       <para>
+        Protect stored cardholder data
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        Encrypt transmission of cardholder data across open, public networks
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/security_pcidss.xml]: this is part of the wording of the standard i think -> keeping the original.

File: xml/security_pcidss.xml [hunk 53]
```diff
@@ -7,272 +7,1251 @@
...
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Maintain a Vulnerability Management Program</term>
+    <listitem>
+     <orderedlist>
+      <listitem override="5">
+       <para>
+        Protect all systems against malware and regularly update anti-virus
+        software or programs
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/security_pcidss.xml]: this is part of the wording of the standard i think -> keeping the original

File: xml/security_pcidss.xml [hunk 54]
```diff
@@ -7,272 +7,1251 @@
...
+   </varlistentry>
+  </variablelist>
   <para>
-   If your organisation develops software intended to be PCI-DSS compliant,
-   you could be obliged to additionally follow the PA-DSS standard. This
-   standard is available from
-   <link xlink:href="https://www.pcisecuritystandards.org/security_standards/documents.php"/>.
+   Most requirements of &pcidssa; are organizational guidelines that
+   help ensure the security of all areas involved with cardholder data.
+   The wording of technical aspects is usually unspecific.
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/security_pcidss.xml]: fxd! thanks for the good proposal.

File: xml/security_pcidss.xml [hunk 55]
```diff
@@ -7,272 +7,1251 @@
...
-  <title>When Your Organization Does Not Need to Be PCI-DSS Compliant</title>
   <para>
-   Often organizations use third-party service providers that
-   handle credit card processing for them. If your organization's equipment
-   comes into contact with cardholder names, expiration dates, and service
-   codes, but not with Primary Account Numbers, there could be no need for
-   your organization to implement PCI-DSS. See also the
-   <link xlink:href="https://www.pcisecuritystandards.org/pdfs/pci_fs_data_storage.pdf">
-   PCI-DSS Data Storage Do's and Don'ts </link>.
+   This means, it is up to auditors to decide which security settings are
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/security_pcidss.xml]: fxd

File: xml/security_pcidss.xml [hunk 56]
```diff
@@ -7,272 +7,1251 @@
...
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     System maintenance to protect against known vulnerabilities
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   Conversely, this document does not concern itself with aspects not within
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/security_pcidss.xml]: fxd

File: xml/security_pcidss.xml [hunk 57]
```diff
@@ -7,272 +7,1251 @@
...
-
+ </sect1>
+ <sect1 xml:id="sec.pcidss.os-relevant">
+  <title>Focus of This Document: Areas Relevant to the Operating System</title>
   <para>
-   This section of the PCI-DSS standard is concerned with enabling,
-   assessing, and maintaining firewall and router configurations. The second
-   part of this section is concerned with default passwords.
+   The &pcidssa; covers many aspects related to cardholder data. Not all of
+   these aspects have bearing on the operating system and this document will
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/security_pcidss.xml]: fxd

File: xml/security_pcidss.xml [hunk 58]
```diff
@@ -7,272 +7,1251 @@
...
-  <title>Securing Networks and Systems</title>
-
+ </sect1>
+ <sect1 xml:id="sec.pcidss.os-relevant">
+  <title>Focus of This Document: Areas Relevant to the Operating System</title>
   <para>
-   This section of the PCI-DSS standard is concerned with enabling,
-   assessing, and maintaining firewall and router configurations. The second
-   part of this section is concerned with default passwords.
+   The &pcidssa; covers many aspects related to cardholder data. Not all of
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/security_pcidss.xml]: fxd

File: xml/security_pcidss.xml [hunk 59]
```diff
@@ -7,272 +7,1251 @@
...
+   </listitem>
+   <listitem>
+    <para>
+     System maintenance to protect against known vulnerabilities
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   Conversely, this document does not concern itself with aspects not within
+   the scope of OS security. For example, this includes data processing
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/security_pcidss.xml]: fxd

File: xml/security_pcidss.xml [hunk 60]
```diff
@@ -7,272 +7,1251 @@
...
+   applications, database design, and formal processes outside of the OS
+   scope. In particular, requirement 9 (restrict physical access) and
+   requirement 12 (maintain a policy) are not discussed by this document.
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/security_pcidss.xml]: done

**Comment by sknorr** [comment]
@dmpop I'd be happy about another review now. (However, I'll admit not having looked as closely at the diffs as perhaps I should have.)

**Pull request merged**